### DISCUSSION TITLE
Create installer for Linux

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#5131] [Platforms]
Can you please provide an installer for Linux/Ubuntu?

Having an executable that doesn't need to be installed is great for portability but I feel it would be a better user experience if Godot were integrated into the system e.g. being able to launch it from the applications menu.

**Comment by ghost** [comment]
Seems like there's already some instructions on creating a .desktop file https://askubuntu.com/a/981445 so this could serve as a start?

**Comment by Calinou** [comment]
You can use the Godot Flatpak, which handles desktop integration automatically (like all Flatpaks): https://flathub.org/apps/details/org.godotengine.Godot
